db/migrate/20180828174803_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,31 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change  
+
+    create_table(:users) do |t|
```

Comment:
this doesn't work if you already have the migration run in a production or testing environment. You can't just do a drop of the database and a new generate.
What we suggest doing is: keep the old migrations as they were and create a new migration adding just the fields required by devise token auth. That way it's safer.
Keep this in mind for the future, it's very rare to change a migration already run in the master branch.

Reply:
Ok, greate.